yarn.lock

```diff
@@ -2764,17 +2764,6 @@ version-guard@^1.1.1:
   resolved "https://registry.npmjs.org/version-guard/-/version-guard-1.1.3.tgz"
   integrity sha512-JwPr6erhX53EWH/HCSzfy1tTFrtPXUe927wdM1jqBBeYp1OM+qPHjWbsvv6pIBduqdgxxS+ScfG7S28pzyr2DQ==
 
```

Comment:
are you using yarn to install your dependencies?

Reply:
Nope, ut was generated by the prev project owner and now it is auto updating

Reply:
kk I'll take care of this